Slicer extension build failure

I'm reviewing extension builds for latest Slicer-4.13 version. This extension fails to build for current Slicer version:

```
CMake Error: Could not find cmake module file: CMakeDetermineFragmentsCompiler.cmake
CMake Error: Could not find cmake module file: D:/D/P/S-0-E-b/SurfaceFragmentsRegistration-build/CMakeFiles/3.22.1/CMakeFragmentsCompiler.cmake
CMake Error: Could not find cmake module file: CMakeDetermineRegistrationCompiler.cmake
CMake Error: Could not find cmake module file: D:/D/P/S-0-E-b/SurfaceFragmentsRegistration-build/CMakeFiles/3.22.1/CMakeRegistrationCompiler.cmake
CMake Error at CMakeLists.txt:3 (project):
  No CMAKE_Fragments_COMPILER could be found.

CMake Error: Could not find cmake module file: CMakeFragmentsInformation.cmake
CMake Error at CMakeLists.txt:3 (project):
  No CMAKE_Registration_COMPILER could be found.
```

(https://slicer.cdash.org/build/2578479)

Would you like to keep this extension available? Do you need help with resolving this error?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.13.4)
 
-project(Surface Fragments Registration)
+project(SurfaceFragmentsRegistration)
 
 #-----------------------------------------------------------------------------
 # Extension meta-information

--- a/SurfaceFragmentsRegistration/Testing/CMakeLists.txt
+++ b/SurfaceFragmentsRegistration/Testing/CMakeLists.txt
@@ -1,2 +1,1 @@
 add_subdirectory(Python)
-add_subdirectory(Assets)

